scripts/setup_irods.py

```diff
@@ -198,7 +198,10 @@ def determine_server_role(irods_config):
     default_catalog_service_role = irods_config.server_config.get('catalog_service_role', 'provider')
     irods_config.server_config['catalog_service_role'] = irods.lib.default_prompt(
         'iRODS server\'s role',
-        default=[default_catalog_service_role] + list(catalog_service_roles - set([default_catalog_service_role])),
+#        default=[default_catalog_service_role] + list(catalog_service_roles - set([default_catalog_service_role])), # remembers, but swaps order
+#        default=[default_catalog_service_role], # remembers, but only text in prompt, no numbered options
+#        default=catalog_service_roles,          # numbered list, always same order, but doesn't remember
+        default=catalog_service_roles, previous=catalog_service_roles.index(default_catalog_service_role) + 1, # numbered list, same order, does remember
```

Comment:
This is definitely my expected behavior. Consider separating these kwargs on separate lines? And should we remove the commented-out lines?

Reply:
yes, will separate and remove comments - those were for review purposes only.

Reply:
done